easybuild/easyconfigs/m/make/make-4.1-GCC-4.9.2.eb

```diff
@@ -0,0 +1,30 @@
+##
+# This file is an EasyBuild recipy as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/
+##
+easyblock = 'ConfigureMake'
+
+name = 'make'
+version = '4.1'
+
+homepage = 'http://www.gnu.org/software/make/make.html'
+description = "GNU version of make utility"
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
```

Comment:
This is 4.9.2, not 4.9.3.

Also, if someone needs this with `GCCcore`, they can add it.